shared/src/main/scala/com/raquo/domtypes/defs/reflectedAttrs/ReflectedHtmlAttrDefs.scala

```diff
@@ -1105,6 +1105,27 @@ object ReflectedHtmlAttrDefs {
       docUrls = Nil,
     ),
 
+    ReflectedHtmlAttrDef(
+      scalaName = "crossorigin",
+      domAttrName = "crossorigin",
+      domPropName = "crossorigin",
```

Comment:
`domPropName` should be camelCased to "crossOrigin" because that's what it's called in JS DOM (it won't work otherwise). I know, it's a bit weird – JS reflected properties can have names that are slightly different from the corresponding attribute names.

Reply:
fixed